### DISCUSSION TITLE
Upcoming Planned items - 6.0.0

### DISCUSSION THREAD

**Discussion by Naveenkumar739** [Sunbird-inQuiry/Community#79] [Announcements]
Hello Folks,

We have kickstarted Release-6.0.0,R 6.0.0 comprises 10 weeks release plan,

Please checkout the plan mentioned below:

|              | Start Date | End Date |
|------------|:-------------:|:---------------|
| Sprint | 24 April 2023 | 19 Jun 2023 |
| PPV     | 23 Jun 2023 | 6 Jul 2023 |
| Prod | 7 July 2023 | |

Following are the features of Inquiry Release 6.0.0
Inquiry Release 6.0.0 - [Link](https://project-sunbird.atlassian.net/issues/?filter=12794&jql=issuetype%20in%20standardIssueTypes()%20AND%20Sprint%20%3D%20421%20order%20by%20created%20DESC)

Do engage with us, if you have any questions.
@kameshbhr @PoojaJManjooran @sajeshkayyath @surendrasinghs @pallakartheekreddy @chitranshu-keshav @krgauraw @rajnishdargan @Aparna-karri591 @vaibhavbhuva @sanyapawah @abhay-samagra @omkar309 @Krishnaj20 @ashokreddy1208 @PoornimaRajesh

Cheers,
InQuiry BB team

**Comment by omkar309** [comment]
Hi @Naveenkumar739 according to this announcement Release 6.0.0 is in progress. But there is also a [calendar sheet](https://docs.google.com/spreadsheets/d/1jEO5gArDZ6tvYdXpQM4VLVITYQaFOjNLG-JcYRIh_Jo/edit#gid=995731450) that I have been referring to, and according to that Release 5.7.0 should be going on right now.

Which is the correct one and which one to follow? And can we get the dates for the cycle that will happen after the current release cycle? It would help us in planning developments.
CC: @abhay-samagra

**Comment by Naveenkumar739** [comment]
Hi @omkar309 

5.7 and 6.0 are going in parallel. Please find the screenshot below for release timeline for 5.7 and 6.0

![image](https://github.com/Sunbird-inQuiry/Community/assets/112618894/48f304b2-9c8f-4de4-b2ea-ca6307fb9fc4)

**Reply by omkar309** [reply]
Noted @Naveenkumar739 . Thanks for the update. And by when can we get the dates for release cycles post 5.7.0/6.0.0?

**Reply by Naveenkumar739** [reply]
5.7.0 - 22 May 2023
6.0.0 - 12 Jun 2023

**Reply by PoojaJManjooran** [reply]
Hi @omkar309, there is an update to the release dates of 6.0.0. Please refer to [this](https://docs.google.com/spreadsheets/d/1HZwE4XXIzQQwiXlnLfEhI-f5hypmaSoEqruAS9PCil0/edit#gid=995731450) sheet.

**Reply by omkar309** [reply]
Hi @PoojaJManjooran, Thanks for the update.

**Comment by Naveenkumar739** [comment]
Hello ,

Currently we are not able to provide the Development Sign Off since we have 1 QA_required and 2 QA_Required_Regression are In development status. Probably those tickets will be release by tomorrow EOD [Link](https://project-sunbird.atlassian.net/issues/?filter=12794&jql=issuetype%20in%20standardIssueTypes()%20AND%20status%20%3D%20%22In%20Development%22%20AND%20Sprint%20%3D%20421%20order%20by%20created%20DESC).

@kameshbhr @PoojaJManjooran @sajeshkayyath @surendrasinghs @pallakartheekreddy @chitranshu-keshav @krgauraw @rajnishdargan @Aparna-karri591 @vaibhavbhuva @sanyapawah @abhay-samagra @omkar309 @Krishnaj20 @ashokreddy1208 @PoornimaRajesh

Cheers,
InQuiry BB team

**Comment by Naveenkumar739** [comment]
R 6.0.0 Sunbird Inquiry - Development Phase Sign Off

Hello All,

This is Development Phase QA validation Sign off for the release 6.0.0 for Sunbird inQuiry Building block.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing Status: Completed
Execution Sheet: [Link](https://docs.google.com/spreadsheets/d/1KIMrMtCseA9yqHvG6Oyc8Z5FRssItzwdnGAwCxcShho/edit#gid=0)
Scope: [Link](https://project-sunbird.atlassian.net/issues/?filter=12794&jql=issuetype%20in%20standardIssueTypes()%20AND%20status%20in%20(Done%2C%20Fixed%2C%20%22In%20Validation%22)%20AND%20labels%20in%20(QA%2C%20QA_Not_Required%2C%20QA_Required%2C%20QA_Required_Regression)%20AND%20Sprint%20%3D%20421%20order%20by%20created%20DESC)
Building Block Sign Off Template: [Link](https://docs.google.com/spreadsheets/d/1KIMrMtCseA9yqHvG6Oyc8Z5FRssItzwdnGAwCxcShho/edit#gid=1425160680)

@kameshbhr @PoojaJManjooran @sajeshkayyath @surendrasinghs @pallakartheekreddy @chitranshu-keshav @krgauraw @rajnishdargan @Aparna-karri591 @vaibhavbhuva @sanyapawah @abhay-samagra @omkar309 @Krishnaj20 @ashokreddy1208 @PoornimaRajesh

**Comment by Naveenkumar739** [comment]
Hello All,

Please find the testing status of R 6.0.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: In Progress (9%)
Planned Regression Completion-Off Date: 30 June 2023

Defect :[Link](https://project-sunbird.atlassian.net/browse/IQ-485)

Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1KIMrMtCseA9yqHvG6Oyc8Z5FRssItzwdnGAwCxcShho/edit#gid=0)

Execution Details:
![image](https://github.com/Sunbird-inQuiry/Community/assets/112618894/a8256133-3507-4a7e-b860-a4d1934deea2)

@kameshbhr @PoojaJManjooran @sajeshkayyath @surendrasinghs @pallakartheekreddy @chitranshu-keshav @krgauraw @rajnishdargan @Aparna-karri591 @vaibhavbhuva @sanyapawah @abhay-samagra @omkar309 @Krishnaj20 @ashokreddy1208 @PoornimaRajesh

**Comment by Naveenkumar739** [comment]
Hello All,

Please find the testing status of R 6.0.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: In Progress (19.91%)
Planned Regression Completion-Off Date: 30 June 2023

Defect :[Link](https://project-sunbird.atlassian.net/browse/IQ-485)

Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1KIMrMtCseA9yqHvG6Oyc8Z5FRssItzwdnGAwCxcShho/edit#gid=0)

Execution Details:
![image](https://github.com/Sunbird-inQuiry/Community/assets/112618894/fa31f0d8-cc4a-4a8b-acea-f47b3d0ec3ac)

@kameshbhr @PoojaJManjooran @sajeshkayyath @surendrasinghs @pallakartheekreddy @chitranshu-keshav @krgauraw @rajnishdargan @Aparna-karri591 @vaibhavbhuva @sanyapawah @abhay-samagra @omkar309 @Krishnaj20 @ashokreddy1208 @PoornimaRajesh

**Comment by Naveenkumar739** [comment]
Hello All,

Please find the testing status of R 6.0.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: In Progress (29.70%)
Planned Regression Completion-Off Date: 30 June 2023

Defect :[Link](https://project-sunbird.atlassian.net/browse/IQ-485)

Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1KIMrMtCseA9yqHvG6Oyc8Z5FRssItzwdnGAwCxcShho/edit#gid=0)

Execution Details:
![image](https://github.com/Sunbird-inQuiry/Community/assets/112618894/ce6a538f-cc13-4781-b590-2d743d33714a)

@kameshbhr @PoojaJManjooran @sajeshkayyath @surendrasinghs @pallakartheekreddy @chitranshu-keshav @krgauraw @rajnishdargan @Aparna-karri591 @vaibhavbhuva @sanyapawah @abhay-samagra @omkar309 @Krishnaj20 @ashokreddy1208 @PoornimaRajesh

**Comment by Naveenkumar739** [comment]
Hello All,

Please find the testing status of R 6.0.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: In Progress (39.49%)
Planned Regression Completion-Off Date: 30 June 2023

Defect :[Link](https://project-sunbird.atlassian.net/browse/IQ-485)

Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1KIMrMtCseA9yqHvG6Oyc8Z5FRssItzwdnGAwCxcShho/edit#gid=0)

Execution Details:
![image](https://github.com/Sunbird-inQuiry/Community/assets/112618894/e68dce32-8938-468e-9719-2b48a46122d1)

@kameshbhr @PoojaJManjooran @sajeshkayyath @surendrasinghs @pallakartheekreddy @chitranshu-keshav @krgauraw @rajnishdargan @Aparna-karri591 @vaibhavbhuva @sanyapawah @abhay-samagra @omkar309 @Krishnaj20 @ashokreddy1208 @PoornimaRajesh

**Comment by Naveenkumar739** [comment]
Hello All,

Please find the testing status of R 6.0.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: In Progress (49.29%)
Planned Regression Completion-Off Date: 6 July 2023

Defect :[Link](https://project-sunbird.atlassian.net/browse/IQ-485)

Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1KIMrMtCseA9yqHvG6Oyc8Z5FRssItzwdnGAwCxcShho/edit#gid=0)

Execution Details:
![image](https://github.com/Sunbird-inQuiry/Community/assets/112618894/8344f6a4-787a-43f2-81c6-dfe8a0999f0e)
@kameshbhr @PoojaJManjooran @sajeshkayyath @surendrasinghs @pallakartheekreddy @chitranshu-keshav @krgauraw @rajnishdargan @Aparna-karri591 @vaibhavbhuva @sanyapawah @abhay-samagra @omkar309 @Krishnaj20 @ashokreddy1208 @PoornimaRajesh

**Comment by Naveenkumar739** [comment]
Hello All,

Please find the testing status of R 6.0.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: In Progress (57.98%)
Planned Regression Completion-Off Date: 6 July 2023

Defect :[Link](https://project-sunbird.atlassian.net/browse/IQ-485)

Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1KIMrMtCseA9yqHvG6Oyc8Z5FRssItzwdnGAwCxcShho/edit#gid=0)

Execution Details:
![image](https://github.com/Sunbird-inQuiry/Community/assets/112618894/63d77088-403a-4a48-a757-febcc392cf18)
@kameshbhr @PoojaJManjooran @sajeshkayyath @surendrasinghs @pallakartheekreddy @chitranshu-keshav @krgauraw @rajnishdargan @Aparna-karri591 @vaibhavbhuva @sanyapawah @abhay-samagra @omkar309 @Krishnaj20 @ashokreddy1208 @PoornimaRajesh

**Comment by Naveenkumar739** [comment]
Hello All,

Please find the testing status of R 6.0.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: In Progress (64.30%)
Planned Regression Completion-Off Date: 6 July 2023

Defect :[Link](https://project-sunbird.atlassian.net/browse/IQ-485)

Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1KIMrMtCseA9yqHvG6Oyc8Z5FRssItzwdnGAwCxcShho/edit#gid=0)

Execution Details:
![image](https://github.com/Sunbird-inQuiry/Community/assets/112618894/dc831e0c-7605-445f-90c8-d0929e5592cf)

@kameshbhr @PoojaJManjooran @sajeshkayyath @surendrasinghs @pallakartheekreddy @chitranshu-keshav @krgauraw @rajnishdargan @Aparna-karri591 @vaibhavbhuva @sanyapawah @abhay-samagra @omkar309 @Krishnaj20 @ashokreddy1208 @PoornimaRajesh

**Comment by Naveenkumar739** [comment]
Hello All,

Please find the testing status of R 6.0.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: In Progress (73.30%)
Planned Regression Completion-Off Date: 6 July 2023

Defect :[Link](https://project-sunbird.atlassian.net/browse/IQ-485)

Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1KIMrMtCseA9yqHvG6Oyc8Z5FRssItzwdnGAwCxcShho/edit#gid=0)

Execution Details:
![image](https://github.com/Sunbird-inQuiry/Community/assets/112618894/faa09e81-1ea4-4e5f-a1bf-f2f57af41571)

@kameshbhr @PoojaJManjooran @sajeshkayyath @surendrasinghs @pallakartheekreddy @chitranshu-keshav @krgauraw @rajnishdargan @Aparna-karri591 @vaibhavbhuva @sanyapawah @abhay-samagra @omkar309 @Krishnaj20 @ashokreddy1208 @PoornimaRajesh

**Comment by Naveenkumar739** [comment]
Hello All,

Please find the testing status of R 6.0.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: In Progress (87.84%)
Planned Regression Completion-Off Date: 6 July 2023

Defect :[Link](https://project-sunbird.atlassian.net/browse/IQ-485)

Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1KIMrMtCseA9yqHvG6Oyc8Z5FRssItzwdnGAwCxcShho/edit#gid=0)

Execution Details:
![image](https://github.com/Sunbird-inQuiry/Community/assets/112618894/a4ca323b-5359-4dd7-b11e-3787ff662697)
@kameshbhr @PoojaJManjooran @sajeshkayyath @surendrasinghs @pallakartheekreddy @chitranshu-keshav @krgauraw @rajnishdargan @Aparna-karri591 @vaibhavbhuva @sanyapawah @abhay-samagra @omkar309 @Krishnaj20 @ashokreddy1208 @PoornimaRajesh

**Comment by Naveenkumar739** [comment]
Hello All,

Please find the testing status of R 6.0.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: In Progress (98.58%)
Planned Regression Completion-Off Date: 6 July 2023

Defect :[Link](https://project-sunbird.atlassian.net/browse/IQ-485)

Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1KIMrMtCseA9yqHvG6Oyc8Z5FRssItzwdnGAwCxcShho/edit#gid=0)

Execution Details:
![image](https://github.com/Sunbird-inQuiry/Community/assets/112618894/3964127c-3abc-4de2-8698-8870dc923ba6)

@kameshbhr @PoojaJManjooran @sajeshkayyath @surendrasinghs @pallakartheekreddy @chitranshu-keshav @krgauraw @rajnishdargan @Aparna-karri591 @vaibhavbhuva @sanyapawah @abhay-samagra @omkar309 @Krishnaj20 @ashokreddy1208 @PoornimaRajesh

**Comment by Naveenkumar739** [comment]
Hello All,

Please find the testing status of R 6.0.0 Sunbird Inquiry Regression validation.

Environment: [Sunbird Dev Inquiry Portal](https://dev.inquiry.sunbird.org/)
Testing completion percentage: Completed(100%)
Planned Regression Completion-Off Date: 6 July 2023

Defect :[Link](https://project-sunbird.atlassian.net/browse/IQ-485)

Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1KIMrMtCseA9yqHvG6Oyc8Z5FRssItzwdnGAwCxcShho/edit#gid=0)

Execution Details:
![image](https://github.com/Sunbird-inQuiry/Community/assets/112618894/3c3f1775-51ca-4569-b5e0-b3574ce94e98)

@kameshbhr @PoojaJManjooran @sajeshkayyath @surendrasinghs @pallakartheekreddy @chitranshu-keshav @krgauraw @rajnishdargan @Aparna-karri591 @vaibhavbhuva @sanyapawah @abhay-samagra @omkar309 @Krishnaj20 @ashokreddy1208 @PoornimaRajesh

**Comment by Naveenkumar739** [comment]
Hello All,

This is Regression Phase QA Sign off for the release 6.0.0 for Sunbird Inquiry Building Block.

Environment: Sunbird Inquiry Portal (https://dev.inquiry.sunbird.org/)
Testing Status: Completed
Execution sheet: [Link](https://docs.google.com/spreadsheets/d/1KIMrMtCseA9yqHvG6Oyc8Z5FRssItzwdnGAwCxcShho/edit#gid=0)
Building Block Sign Off Template: [Link](https://docs.google.com/spreadsheets/d/1KIMrMtCseA9yqHvG6Oyc8Z5FRssItzwdnGAwCxcShho/edit#gid=1425160680)
Inquiry BB Microsite for Release Notes: [Link](https://inquiry.sunbird.org/use/release-notes/inquiry-release-v6.0.0-ongoing)
cc: @kameshbhr @PoojaJManjooran @sajeshkayyath @surendrasinghs @pallakartheekreddy @chitranshu-keshav @krgauraw @rajnishdargan @Aparna-karri591 @vaibhavbhuva @sanyapawah @abhay-samagra @omkar309 @Krishnaj20 @ashokreddy1208 @PoornimaRajesh
